src/util.js

```diff
@@ -106,3 +107,127 @@ export function format_memory_and_limit(usage, limit) {
         return _("");
     }
 }
+
+// TODO
+export function container_stop(container, timeout) {
+}
+
+export function updateContainers() {
+    let newContainers = [];
+    let newContainersStats = [];
+    return new Promise((resolve, reject) => {
+        varlinkCall(PODMAN, "io.podman.ListContainers")
+                .then(reply => {
+                    let newContainersMeta = reply.containers;
+                    if (!newContainersMeta) {
+                        resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                        return;
+                    }
+                    const len = newContainersMeta.length;
```

Comment:
This isn't being used anywhere near. If it is two hundred lines below, everyone will have forgotten whose length `len` is. So please drop that and spell it out where you need it, that's safer.